src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -159,12 +159,23 @@ public string GetActiveBufferName()
             return GetDTE().ActiveDocument.Name;
         }
 
+        public string GetActiveWindowName()
```

Comment:
Does this only do tool windows? Or what does it return if the active window is a document window?